src/components/Cars/AddCar.jsx

```diff
@@ -0,0 +1,125 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import axios, { AxiosError } from 'axios';
+import { useAuthHeader, useAuthUser } from 'react-auth-kit';
+
+function AddCar() {
+  const auth = useAuthUser();
+  const [info, setInfo] = useState({
+    name: '',
+    description: '',
+    image: '',
+    classification: '',
+    model: '',
+    year: '',
+    price_per_day: '',
+    user_id: auth().id,
+  });
+  const [error, setError] = useState('');
+  const navigate = useNavigate();
+  const authHeader = useAuthHeader();
+
+  const changeHandler = (e) => {
+    const { name, value } = e.target;
+    setInfo({ ...info, [name]: value });
+  };
+
+  const handleSubmit = async () => {
+    setError('');
+
+    try {
+      const res = await axios.post('http://localhost:3000/api/v1/cars', info, {
+        headers: {
+          Authorization: authHeader(),
+        },
+      });
+      if (res.status === 200) {
+        navigate('/cars');
+      }
```

Comment:
A successful form submission does not currently redirect to the cars page. I believe the problem may be that the submit function is missing a `preventDefault()` call. Please take a look at this